.github/workflows/tests.yml

```diff
@@ -276,6 +276,9 @@ jobs:
       - name: Install openfermionpyscf
         run: pip3 install openfermionpyscf
 
+      - name: Install basis-set-exchange

```

Comment:
I don't understand what exactly was the problem with PubChem, but let's make sure we don't run into the same issues with basis-set-exchange

Reply:
Loading molecular geometries from PubChem needs access to the PubChem data server.  For basis-set-exchange, installing the package is enough.